src/actions/index.js

```diff
@@ -11,3 +11,4 @@ export {
 export { USER_ACTION_TYPES, UserActions } from './UserActions';
 export { CASH_TRANSACTION_ACTION_TYPES, CashTransactionActions } from './CashTransactionActions';
 export { DASHBOARD_ACTION_TYPES, DashboardActions } from './DashboardActions';
+export { VACCINE_ACTIONS, VaccineActions } from './VaccineActions';
```

Comment:
I was a little confused by this file because there's a lot of actions that aren't here 🤔   but seems like a good idea to have them all indexed.